core/src/main/kotlin/model/Documentable.kt

```diff
@@ -93,12 +108,14 @@ class BasePlatformInfo(
 
 class ClassPlatformInfo(
     val info: PlatformInfo,
-    val inherited: List<DRI>) : PlatformInfo by info
+    val inherited: List<DRI>
+) : PlatformInfo by info
 
 abstract class Documentable {
     open val expected: PlatformInfo? = null
     open val actual: List<PlatformInfo> = emptyList()
     open val name: String? = null
+    open val sourceLocation: String? = null
```

Comment:
I'm not sure if we need this info there as it looks platform-specific. In any case, it should not be singular

Reply:
Removed.